chef_master/source/windows.rst

```diff
@@ -3,181 +3,144 @@ Chef for Microsoft Windows
-
-The most popular core resources in the Chef Infra Client---`cookbook_file </resource_cookbook_file.html>`__, `directory </resource_directory.html>`__, `env </resource_env.html>`__, `execute </resource_execute.html>`__, `file </resource_file.html>`__, `group </resource_group.html>`__, `http_request </resource_http_request.html>`__, `link </resource_link.html>`__, `mount </resource_mount.html>`__, `package </resource_package.html>`__, `remote_directory </resource_remote_directory.html>`__, `remote_file </resource_remote_file.html>`__, `ruby_block </resource_ruby_block.html>`__, `service </resource_service.html>`__, `template </resource_template.html>`__, and `user </resource_user.html>`__---work the same way in Microsoft Windows as they do on any UNIX- or Linux-based platform.
-
-The file-based resources---**cookbook_file**, **file**, **remote_file**, and **template**---have attributes that support unique requirements within the Microsoft Windows platform, including ``inherits`` (for file inheritance), ``mode`` (for octal modes), and ``rights`` (for access control lists, or ACLs).
-
-.. note:: The Microsoft Windows platform does not support running as an alternate user unless full credentials (a username and password or equivalent) are specified.
-
-The following sections are pulled in from the larger |url docs| site and represents the documentation that is specific to the Microsoft Windows platform, compiled here into a single-page reference.
-
-Install the Chef Infra Client on Windows
-=====================================================
 .. tag windows_install_overview
 
 The Chef Infra Client can be installed on machines running Microsoft Windows in the following ways:
 
-* By using `knife windows </knife_windows.html>`__ to bootstrap the Chef Infra Client; this process requires the target node be available via the WinRM port (typically port 5985)
-* By downloading the Chef Infra Client to the target node, and then running the Microsoft Installer Package (MSI) locally
+* By using `knife windows </knife_windows.html>`__ to bootstrap Chef Infra Client; this process requires the target node be available via the WinRM port (typically port 5985)
+* By downloading Chef Infra Client to the target node, and then running the Microsoft Installer Package (MSI) locally
 * By using an existing process already in place for managing Microsoft Windows machines, such as System Center
 
-To run the Chef Infra Client at periodic intervals (so that it can check in with the Chef Infra Server automatically), configure the Chef Infra Client to run as a service or as a scheduled task. (The Chef Infra Client can be configured to run as a service during the setup process.)
+To run Chef Infra Client at periodic intervals (so that it can check in with Chef Infra Server automatically), configure Chef Infra Client to run as a service or as a scheduled task. (The Chef Infra Client can be configured to run as a service during the setup process.)
```

Comment:
Let's just strike the bit about running as a service. We really don't want people to do that.

Reply:
Also this:
```
Enable as a Scheduled Task
+++++++++++++++++++++++++++++++++++++++++++++++++++++
.. tag install_chef_client_windows_as_scheduled_task

To run Chef Infra Client at periodic intervals (so that it can check in with Chef Infra Server automatically), configure Chef Infra Client to run as a scheduled task. This can be done via the MSI, by selecting the **Chef Unattended Execution Options** --> **Chef Infra Client Scheduled Task** option on the **Custom Setup** page or by running the following command after Chef Infra Client is installed:

For example:

.. code-block:: none

   $ SCHTASKS.EXE /CREATE /TN ChefClientSchTask /SC MINUTE /MO 30 /F /RU "System" /RP /RL HIGHEST /TR "cmd /c \"C:\opscode\chef\embedded\bin\ruby.exe C:\opscode\chef\bin\chef-client -L C:\chef\chef-client.log -c C:\chef\client.rb\""

Refer `Schedule a Task <https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc748993(v=ws.11)>`_ for more details.

After Chef Infra Client is configured to run as a scheduled task, the default file path is: ``c:\chef\chef-client.log``.

.. end_tag
```